spring-boot-project/spring-boot-actuator-autoconfigure/src/test/java/org/springframework/boot/actuate/autoconfigure/tracing/prometheus/PrometheusExemplarsAutoConfigurationTests.java

```diff
@@ -57,10 +58,10 @@ class PrometheusExemplarsAutoConfigurationTests {
 	private final ApplicationContextRunner contextRunner = new ApplicationContextRunner()
 		.withPropertyValues("management.tracing.sampling.probability=1.0",
 				"management.metrics.distribution.percentiles-histogram.all=true")
-		.with(MetricsRun.limitedTo(PrometheusMetricsExportAutoConfiguration.class))
-		.withConfiguration(
-				AutoConfigurations.of(PrometheusExemplarsAutoConfiguration.class, ObservationAutoConfiguration.class,
-						BraveAutoConfiguration.class, MicrometerTracingAutoConfiguration.class));
+		.with(MetricsRun.limitedTo())
+		.withConfiguration(AutoConfigurations.of(PrometheusSimpleclientMetricsExportAutoConfiguration.class,
```

Comment:
This was done to avoid adding the deprecated `PrometheusSimpleclientMetricsExportAutoConfiguration` to `MetricsRun`. When exemplar support is fixed for the latest Prometheus client (planned to be in time for Micrometer 1.13.0-RC1), this can be updated to use `PrometheusMetricsExportAutoConfiguration`. But if we want to continue testing exemplar support with the simpleclient module, this test class will need to be duplicated with this same setup.